js/collapse.js

```diff
@@ -16,14 +16,10 @@
   var Collapse = function (element, options) {
     this.$element      = $(element)
     this.options       = $.extend({}, Collapse.DEFAULTS, options)
-    this.$trigger      = $(this.options.trigger).filter('[href="#' + element.id + '"], [data-target="#' + element.id + '"]')
+    this.$allTriggers  = $('[data-toggle="collapse"]').filter('[href="#' + this.$element[0].id + '"], [data-target="#' + this.$element[0].id + '"]')
```

Comment:
Perhaps we should compute this on-demand, so that it doesn't become outdated when folks add/delete triggers after the Collapse gets initialized?